test/DynamicDependency/Test_WinRT/TestCom.cpp

```diff
@@ -0,0 +1,41 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+#include "pch.h"
```

Comment:
this is a total dup of another exactly same file?  central header that everyone can use?

Reply:
Yes. There's some internal shuffling I plan to make in a follow up PR for things like this.

Common code poses a certain tax (esp over time) so we need to be careful not to cause more harm than help. That said, there do seem to be some things worth sharing. I focused on getting DynDep working and deferred any "commonalize useful bits" until both my work stabilized and the solution overall was grounded.